assert/assert.go

```diff
@@ -90,6 +90,14 @@ func (assert Assert) EqFloat(actual, expected float64) {
 	}
 }
 
+// EqFloat fails the test if two floats aren't equal. NaNs are considered equal.
+func (assert Assert) EqApproximate(actual, expected, epsilon float64) {
+	delta := actual - expected
+	if !(-epsilon < delta && delta < epsilon) {
```

Comment:
It would mean that you couldn't use `.ExApproximate` if you've got a testcase whose results could possibly be `NaN` or `Inf` (and are expected) since they'd fail no matter what was returned, and you'd have to split into another test case. This wouldn't be a problem with single-value returns, but when returning a list whose results are approximate (but might contain `NaN`) then you wouldn't be able to use this function (alone).